at-tanvo/D3-Exercise2.rb

```diff
@@ -0,0 +1,11 @@
+puts "Nhap phan tu mang :"
+a = gets.to_i
+arr = []
+for i in 1..a
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/5c4b6527b3afcb714c4f0ab1934ff6c19402ac0b/at-tanvo/D3-Exercise2.rb#L4"></span>Style/For: Prefer <code>each</code> over <code>for</code>. (https://github.com/rubocop-hq/ruby-style-guide#no-for-loops)</td>
    </tr>
  </tbody>
</table>